proxy/plugin/reverse_proxy.py

```diff
@@ -49,7 +53,11 @@ def routes(self) -> List[Union[str, Tuple[str, List[bytes]]]]:
             r'/get/(\d+)$',
         ]
 
-    def handle_route(self, request: HttpParser, pattern: RePattern) -> Url:
+    def handle_route(
+        self,
+        request: HttpParser,
+        pattern: RePattern,
+    ) -> Union[memoryview, Url, 'TcpServerConnection']:
```

Comment:
Handle route can now return

1) Url (as before)
2) memory view for direct response serving
3) A TcpServerConnection instance